### PR TITLE
New celery beat for LMS tasks

### PR DESCRIPTION
**Pull request by marcospri**
Needs https://github.com/hypothesis/lms/pull/4142 merged first

`LMS_BROKER_URL` already set on prod/qa and Canada environments.

# Testing

- Checkout https://github.com/hypothesis/lms/pull/4142 on LMS and `make services dev`
- Change the perodicity of the task here to something like `minutes=1`, `make dev`
- See the executing of the task on LMS's `make dev` logs.

### CODE REVIEW

File: tests/unit/h_periodic/lms_beat_test.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# pylint: disable=unused-import
+# This is for some dodgy test coverage
```
Comment (marcospri) [inline tests/unit/h_periodic/lms_beat_test.py:2]: Copied this from the checkmate equivalent. 

This is probably a little bit better tha ignoring the file if the config file (ie if we add more functions here coverage won't be complete).

File: h_periodic/lms_beat.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            routing_key="celery",
+            exchange=Exchange("celery", type="direct", durable=False),
+        ),
+    ],
+    beat_schedule_filename="lms-celerybeat-schedule",
+    beat_schedule={
+        "rotate-rsa-keys": {
+            "options": {"expires": 600},
+            "task": "lms.tasks.tasks.rotate_keys",
+            "schedule": timedelta(days=1),
```
Comment (marcospri) [inline h_periodic/lms_beat.py:33]: Run this once a day.

File: conf/supervisord.conf [hunk 3]
```diff
@@ -18,6 +18,13 @@ stderr_events_enabled=true
 stopsignal = KILL
 stopasgroup = true
 
+[program:checkmate-beat]
```
Comment (jon-betts) [inline conf/supervisord.conf:21]:
```suggestion
[program:lms-beat]
```
?

File: README.markdown [hunk 4]
```diff
@@ -51,6 +51,8 @@ linting, code formatting, etc.
 |----------------------|-------|---------|
 | `H_BROKER_URL`         | The `h` AMPQ broker | `amqp://user:password@rabbit.example.com:5672//` |
 | `CHECKMATE_BROKER_URL` | The `checkmate` AMPQ broker | `amqp://user:password@rabbit.example.com:5673//` |
+| `LMS_BROKER_URL` | The `LMS` AMPQ broker | `amqp://user:password@rabbit.example.com:5674//` |
 | `DISABLE_H_BEAT` | Whether to disable the `h_beat` process | `true` to disable the `h_beat` process, `false` to leave it enabled. Defaults to `false` (leave it enabled) |
 | `DISABLE_CHECKMATE_BEAT` | Whether to disable the `checkmate_beat` process | `true` to disable the `checkmate_beat` process, `false` to leave it enabled. Defaults to `false` (leave it enabled) |
```
Comment (jon-betts) [inline README.markdown:56]: A line here about `DISABLE_LMS_BEAT`?

File: .gitignore [hunk 5]
```diff
@@ -2,11 +2,7 @@
...
 .coverage
 .coverage.*
-h-celerybeat-schedule
-h-celerybeat-schedule.*
-h-celerybeat.pid
-checkmate-celerybeat-schedule
-checkmate-celerybeat-schedule.*
-checkmate-celerybeat.pid
+*-celerybeat-schedule.*
+*-celerybeat.pid
```
Comment (jon-betts) [inline .gitignore:6]: :+1:

**Review by jon-betts — CHANGES_REQUESTED** [review]
The local config works, but I think there's a typo in the live config.

It would also be nice to document the new environment variable `DISABLE_LMS_BEAT`.

**Pull request merged**